tests/benchmarks/test_fastapi_startup_generics.py

```diff
@@ -132,7 +133,9 @@ def bench():
     print(f'Python version: {sys.version}')
     if sys.argv[-1] == 'cProfile':
         cProfile.run(
-            'test_fastapi_startup_perf(lambda f: f())', sort='tottime', filename=__file__.strip('.py') + '.cprof'
+            'test_fastapi_startup_perf(lambda f: f())',
+            sort='tottime',
+            filename=Path(__file__).name.strip('.py') + '.cprof',
```

Comment:
Misc improvement unrelated to this PR, happy to move to another one if requested.